app/models/concerns/md_residence_county_concern.rb

```diff
@@ -3,7 +3,6 @@ module MdResidenceCountyConcern
 
   COUNTIES_AND_SUBDIVISIONS = {
     "Allegany" => {
```

Comment:
i would probably also change the logic in md502 xml where it says `      unless @intake.political_subdivision&.end_with?("- unincorporated")
        xml.CityTownOrTaxingArea @intake.political_subdivision
      end`